children/factories.py

```diff
@@ -0,0 +1,29 @@
+import factory
+from django.contrib.auth.hashers import make_password
+from factory.random import randgen
+
+from children.models import Child, Relationship
+from users.factories import UserFactory
+
+
+class ChildFactory(factory.django.DjangoModelFactory):
+    first_name = factory.Faker("first_name")
+    last_name = factory.Faker("last_name")
+    birthdate = factory.Faker("date_this_year", before_today=True, after_today=False)
+    social_security_number_hash = factory.LazyFunction(
+        lambda: make_password(randgen.random())
+    )
+
+    class Meta:
+        model = Child
+
+
+class RelationshipFactory(factory.django.DjangoModelFactory):
+    child = factory.SubFactory(ChildFactory)
+    user = factory.SubFactory(UserFactory)
+    type = factory.LazyFunction(
+        lambda: randgen.choice([c[0] for c in Relationship.TYPE_CHOICES])
```

Comment:
would this work here:
```python
type = factory.fuzzy.FuzzyChoice(Relationship.TYPE_CHOICES)
```

Reply:
or probably:
```python
RELATIONSHIP_TYPES = [t[0] for t in Relationship.TYPE_CHOICES]
type = factory.fuzzy.FuzzyChoice(RELATIONSHIP_TYPES)
```

Reply:
or maybe leave it be 😅

Reply:
I think the first one would work, but the problem with those `factory.fuzzy.xxx` is that they seem to be more or less deprecated according to [this](https://factoryboy.readthedocs.io/en/latest/fuzzy.html).

Reply:
ah, nice link. so, we can do:
```
type = factory.Faker('random_choice', choices=Relationship.TYPE_CHOICES)
```

Reply:
or actually:
```python
RELATIONSHIP_TYPES = [t[0] for t in Relationship.TYPE_CHOICES]
type = factory.Faker("random_element", elements=RELATIONSHIP_TYPES)
```